src/Codec/Xlsx/Parser.hs

```diff
@@ -700,9 +700,17 @@ getRels ar fp = do
     c <- xmlCursorOptional ar relsPath
     return $ maybe Relationships.empty (setTargetsFrom fp . headNote "Missing rels" . fromCursor) c
 
+-- According to part 2, section 7.3.4 of ECMA-376, when mapping logical item
+-- names to ZIP item names we need to remove the leading slash.
+--
+-- <http://www.ecma-international.org/publications/standards/Ecma-376.htm>
```

Comment:
This link looks to be outdated - it returns 404 for me.
The new Url appears to be https://ecma-international.org/publications-and-standards/standards/ecma-376/

Reply:
Updated, sorry.

Reply:
np, I guess I should also update the link in cabal